huaweicloud/services/acceptance/aom/resource_huaweicloud_aom_alarm_inhibit_rule_test.go

```diff
@@ -0,0 +1,545 @@
+					resource.TestCheckResourceAttr(rNameWithoutGroupRule, "target_matches.#", "0"),
+				),
+			},
+			{
+				Config: testAccAlarmInhibitRule_with_matchv3AndEpsId_step2(name),
+				Check: resource.ComposeTestCheckFunc(
+					rcWithGroupRule.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rNameWithGroupRule, "binding_group_rule", ""),
+					rcWithoutGroupRule.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rNameWithoutGroupRule, "binding_group_rule",
+						"huaweicloud_aom_alarm_group_rule.test", "name"),
+				),
+			},
+			{
+				ResourceName:      "huaweicloud_aom_alarm_inhibit_rule.test[0]",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccAlarmInhibitRuleWithEpsIdImportStateFunc(rNameWithGroupRule),
+			},
+			{
+				ResourceName:      "huaweicloud_aom_alarm_inhibit_rule.test[1]",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccAlarmInhibitRuleWithEpsIdImportStateFunc(rNameWithoutGroupRule),
+			},
+		},
+	})
+}
+
+func testAccAlarmInhibitRule_with_matchv3AndEpsId_step1(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "huaweicloud_aom_alarm_inhibit_rule" "test" {
+  count = 2
+
+  name                  = "%[2]s${count.index}"
+  enterprise_project_id = "%[3]s"
+  binding_group_rule    = count.index == 0 ? huaweicloud_aom_alarm_group_rule.test.name : null
+
+  match_v3 = jsonencode({
+    nodes = [
+      {
+        id   = "flow-VHSvKyYY"
+        type = "Start"
+      },
+      {
+        id           = "flow-ufs5Zsn3"
+        type         = "Card"
+        businessType = "InhibitSourceCondition"
+        value = {
+          id    = "InhibitSourceCondition-wtJBNJD7"
+          type  = "bool"
+          value = "and"
+          nodes = [
+            {
+              id   = "InhibitSourceCondition-awPpHgHv"
+              type = "text"
+              match = {
+                key     = "resource_type"
+                operate = "EQUALS"
+                value   = ["service"]
+              }
+            },
+            {
+              id   = "InhibitSourceCondition-DmfXe2vN"
+              type = "text"
+              match = {
+                key     = "resource_type"
+                operate = "EQUALS"
+                value   = ["service"]
+              }
+            },
+            {
+              id    = "InhibitSourceCondition-0fnYAVWg"
+              type  = "bool"
+              value = "and"
+              nodes = [
+                {
+                  id   = "InhibitSourceCondition-Xn7CUof5"
+                  type = "text"
+                  match = {
+                    key     = "event_severity"
+                    operate = "EQUALS"
+                    value   = ["Critical"]
+                  }
+                }
+              ]
+            },
+            {
+              id   = "InhibitSourceCondition-juUOsA2L"
+              type = "text"
+              match = {
```

Comment:
```suggestion
              id    = "InhibitSourceCondition-juUOsA2L"
              type  = "text"
              match = {
```
check the other places

Reply:
done